### ISSUE TITLE
Option to always display alt text

### ISSUE THREAD

**Issue by maltfield** [idunnololz/summit#50]
This issue is a feature request to add a new option named "always display alt text"

# Problem

Visually impaired (or blind) users need to be able to read the alt text when they come across an image in Summit.

Also, alt texts can display useful additional information about an image for able-eyed folks.

But, currently, it's not clear if an image has alt text. Or how to view it.

# Solution

A new option should be added to the settings of Summit, titled "always display alt text"

If this option is enabled, then a "caption" should be displayed below every image, which shows the contents of the alt text for the image.

**Comment by idunnololz** [comment]
Tracked on the roadmap: https://lemmy.world/post/41317734
